docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

```diff
@@ -0,0 +1,318 @@
+[[conditions-based-autodiscover]]
+= Conditions based autodiscover
+
+You can define autodiscover conditions in each input to allow {agent} to automatically identify Pods and start monitoring them using predefined integrations. You can find more about these inputs and their configurations <<elastic-agent-input-configuration,here>>.
+
+== Example: Target pods by host and label
+
+To automatically identify a Redis Pod and monitor it with the Redis integration, uncomment the following input configuration inside  the {agent} Standalone manifest [here](https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml):
+
+
+[source,yaml]
+------------------------------------------------
+- name: redis
+  type: redis/metrics
+  use_output: default
+  meta:
+    package:
+      name: redis
+      version: 0.3.6
+  data_stream:
+    namespace: default
+  streams:
+    - data_stream:
+        dataset: redis.info
+        type: metrics
+      metricsets:
+        - info
+      hosts:
+        - '${kubernetes.pod.ip}:6379'
+      idle_timeout: 20s
+      maxconn: 10
+      network: tcp
+      period: 10s
+      condition: ${kubernetes.labels.app} == 'redis'
+------------------------------------------------
+
+The condition `${kubernetes.labels.app} == 'redis'` will make the {agent} look for a Pod with the `redis` label app within the scope defined in its manifest. 
+
+List of exported fields of the Kubernetes integration {metricbeat-ref}/exported-fields-kubernetes-processor.html[here] to be used in conditions.
+
+The `redis` input defined in the {agent} manifest only has `info` as metricset. You can find more metricsets in the {metricbeat-ref}/metricbeat-module-redis.html[Redis module page], as well as the configuration options.
+
+To deploy Redis, you can apply the following example manifest:
+
+[source,yaml]
+------------------------------------------------
+apiVersion: v1
```

Comment:
Do we need all these extra stuff? Maybe a simpler manifest fits better to the docs. 
I use the following to deploy a simple Redis Pod: 
```yml
apiVersion: v1
kind: Pod
metadata:
  name: redis
  annotations:
    #co.elastic.metrics/module: redis
    #co.elastic.metrics/hosts: '${data.host}:6379'
    
    co.elastic.hints/package: redis
    co.elastic.hints/host: '${kubernetes.pod.ip}:6379'
    co.elastic.hints/data_streams: info, key
    #co.elastic.hints/info.period: 5s
  labels:
    k8s-app: redis
    app: redis
spec:
  containers:
  - image: redis
    imagePullPolicy: IfNotPresent
    name: redis
    ports:
    - name: redis
      containerPort: 6379
      protocol: TCP
```